# LINCOLN MKZ Parameters
lookAhead: 5.0
wheelBase: 2.85
steeringRatio: 14.8
steerLim:
 lower: -8.4
 upper: 8.4
# PID GAINS
# large error throttle
PID1:
 P: 0.005
 I: 0.005
 D: 0.005
# large error brake
PID2:
 P: 0.05
 I: 0.
 D: 0.
# small error brake
PID3:
 P: 0.05
 I: 0.
 D: 0.
# Tolerance
Tol: 0.5
Adaptive:
 lMin: 5.
 lMax: 7.
 vMin: 3
 vMax: 8
 gamma: 0.75
 ayLim: 0.5


